### PR TITLE
Wfnews 2007

### PR DESCRIPTION
**Pull request by yzlucas**

### CODE REVIEW

File: client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.ts [hunk 1]
```diff
@@ -242,4 +272,16 @@ export class EvacAlertFullDetailsComponent implements OnInit {
       );
     }
   }
+
+  fixPolygonToMap(polygonData,response?) {
+    const convex = this.commonUtilityService.createConvex(polygonData);
+    const bounds = convex.reduce((acc, coord) => [
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.ts:278]: Would be a good idea to add a null check for convex here

File: client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.ts [hunk 2]
```diff
@@ -61,16 +66,41 @@ export class EvacAlertFullDetailsComponent implements OnInit {
...
+      this.eventNumber,
+      {
+        returnGeometry: true,
+      },
+    )
+    .toPromise()
+    .then((response) => {
+        if (response?.features?.length > 0 && response?.features[0].geometry?.rings?.length > 0){
+          const polygonData = this.commonUtilityService.extractPolygonData(response.features[0].geometry.rings);
+          if (polygonData.length) {
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/full-details/evac-alert-full-details/evac-alert-full-details.component.ts:81]: A null check/safe operator for polygonData would also be good

**Review by ssylver93 — APPROVED** [review]
A couple of minor comments for safe operators but looks good otherwise

**Comment by yzlucas** [comment]
> Is this the correct ticket number? These changes do not seem to be related to text wrapping.

Sorry, WFNEWS-1947

**Pull request merged**